standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -589,6 +589,9 @@ public static ConfVars getMetaConf(String name) {
             "in which a warning is being raised if multiple worker version are detected.\n" +
             "The setting has no effect if the metastore.metrics.enabled is disabled \n" +
             "or the metastore.acidmetrics.thread.on is turned off."),
+    COMPACTOR_METADATA_CACHE_TIMEOUT("metastore.compactor.metadata.cache.timeout",
+      "hive.metastore.compactor.metadata.cache.timeout", 60, TimeUnit.SECONDS,
+      "Number of seconds the table/partition metadata are cached by the compactor. Setting it to zero disables the feature."),
```

Comment:
This is only table level

Reply:
Missed this, when removed partition caching. Fixed, thanks.